pkg/cloud/services/ec2/console_test.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ec2
+
+import (
+	"encoding/base64"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/golang/mock/gomock"
+
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/awserrors"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/scope/mocks"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/services/ec2/mock_ec2iface"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/services/ssm/mock_ssmiface"
+)
+
+func TestService_GetConsoleOutput(t *testing.T) {
```

Comment:
nit: Test case naming convention can be taken care of.

Reply:
@shivi28 are we not going to take care of this? Just a reminder as didn't receive any comment here

Reply:
I thought you have referred to the name of test function as you have given comment on that line i.e I have changed the function name from `TestService_GetConsoleOutput ` to `TestGetConsoleOutput`.

But I will definitely make the changes as requested

Reply:
my mistake @shivi28 , I think I have given you link here :) https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/3099#discussion_r790509897

Reply:
No issues Ankita, thanks for the [reference](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea). I will do the requested change